src/GuiBase/Viewer/Viewer.cpp

```diff
@@ -445,12 +450,12 @@ void Gui::Viewer::showEvent( QShowEvent* ev ) {
         auto light =
             new Engine::DirectionalLight( Ra::Engine::SystemEntity::getInstance(), "headlight" );
         // Set the default light intensity so that it does not burn the objects
-        light->setColor( Ra::Core::Color( 0.5, 0.5, 0.5, 1.0 ) );
+        light->setColor( Ra::Core::Utils::Color::Grey( Scalar( 0.5 ) ) );
```

Comment:
this should now be left as default `White()`, according to commit `b73775065b14c5a1a465827e51aa28c15fc95541`

Reply:
This will be fixed later for v2, or next PR by @MathiasPaulin